echo input 1 then 2
/scratch/ejdennis/cm2_brains/m128/642
smartspim
QStandardPaths: XDG_RUNTIME_DIR points to non-existing path '/run/user/2333', please create it with 0700 permissions.
libEGL warning: DRI2: failed to create any config
libEGL warning: DRI2: failed to create any config
libEGL warning: DRI2: failed to create any config
libEGL warning: DRI2: failed to create any config
/usr/people/ejdennis/.conda/envs/cm2/lib/python3.6/site-packages/vispy/visuals/line/line.py:395: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  ('color', np.float32, 4)])
/usr/people/ejdennis/.conda/envs/cm2/lib/python3.6/site-packages/vispy/visuals/line/arrow.py:57: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  ('linewidth', np.float32, 1)
/usr/people/ejdennis/.conda/envs/cm2/lib/python3.6/site-packages/vispy/visuals/isocurve.py:22: UserWarning: VisPy is not yet compatible with matplotlib 2.2+
  warnings.warn("VisPy is not yet compatible with matplotlib 2.2+")
about to import workspace
starting
sysarg v output is ['cell_detect.py', '3', '/scratch/ejdennis/cm2_brains/m128/642', 'smartspim']
/n params are k5, 130
Workspace[CellMap]{/scratch/ejdennis/cm2_brains/m128/642}
              raw: Z<Z,4>.tif {3785 files, ('Z',): (0,) -> (3784,)}
 autofluorescence: no file
         stitched: no file
           layout: no file
       background: no file
        resampled: no file
resampled_to_auto: no file
auto_to_reference: no file
            cells: no file
          density: no file

None
Traceback (most recent call last):
  File "cell_detect.py", line 122, in <module>
    final_results = np.vstack([np.hstack(r) for r in block_result_list]) # merges results into a single array
  File "<__array_function__ internals>", line 6, in vstack
  File "/usr/people/ejdennis/.conda/envs/cm2/lib/python3.6/site-packages/numpy/core/shape_base.py", line 283, in vstack
    return _nx.concatenate(arrs, 0)
  File "<__array_function__ internals>", line 6, in concatenate
ValueError: need at least one array to concatenate
